probe-rs/src/bin/probe-rs/cmd/dap_server/debug_adapter/dap/repl_commands.rs

```diff
@@ -361,4 +361,79 @@ pub(crate) static REPL_COMMANDS: &[ReplCommand<ReplHandler>] = &[
             memory_read(input_address, gdb_nuf, target_core)
         },
     },
+    ReplCommand {
+        command: "dump",
```

Comment:
There is a similar command in the cli debug adapter. We should keep them compatible, and not create two incompatible core dump mechanism.

Reply:
Yup, I updated that in a next commit already which I did not push yesterday. I am already trying to use it for testing the stacktrace function :)
The vscode REPL command is nice to dump a specific state when debugging with VSCode.